client/package.json

```diff
@@ -37,6 +37,8 @@
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
     "seamless-immutable": "^7.1.4",
+    "uuid": "^8.3.1",
+    "uuidv4": "^6.2.5",
```

Comment:
why are we using 2 packages just for generate uuid ?

Reply:
addressed